src/server/types.ts

```diff
@@ -253,6 +253,22 @@ export type RemoveOldRemoteDrivesOption =
     | {
```

Comment:
Can you had a short description to each strategy?

Reply:
added here [181709b](https://github.com/powerhouse-inc/document-drive/pull/305/commits/181709bd471ba6838d8e03834c00fe9a490191bb)